Docs subteam handbook updates (#2383)
kubernetes/sig-release: release-team/role-handbooks/docs/Release-Timeline.md

⚠️  Attend [SIG Docs meetings](https://github.com/kubernetes/community/tree/master/sig-docs#meetings) to provide a status update. 
If you cannot attend, a Shadow needs to attend in your place and/or update the [SIG Docs meeting notes](https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/edit?usp=sharing) with the information necessary to keep the community informed (e.g. staffing updates, branch syncs, placeholder PR deadlines, etc).
- Here is a checklist to guide the weekly status report. 

_Note:_ SIG Docs prefers and welcomes a status of yellow or red on anything that needs attention, has the possiblity to miss deadlines or target dates, or requires follow-up from feature authors and SIGs that you're not receiving.

[ ] Is the integration branch healthy (does not need rebase)?

[ ] Has the weekly branch sync been done? (a lapse in 1-2 branch syncs indicates 'yellow' status, 3 or more lapses should be deemed red)

> [ ] Has the PR for the weekly branch sync merged or needs review? If assistance is needed to resolve merge conflicts, please flag others by communicating on the #release-docs Slack channel

Reviewer: ```suggestion
[ ] Has the PR for the weekly branch sync merged or does it need review? If assistance is needed to resolve merge conflicts, please flag others by communicating on the #release-docs Slack channel
```
Minor nit

[ ] A week before the Docs PR Placeholder deadline, have 80% of the enhancements that opted in for docs created a placeholder PR? (anything below 60% opt-in for docs where placeholder PRs are needed should indicate yellow (or red) status, contingent on the amount)

> [ ] Three days before the Docs PR Placeholder deadline, have 90% of the enhancements that opted in for docs created a placeholder PR?

Reviewer: ```suggestion
[ ] Three days before the Docs PR Placeholder deadline, have 90% of the enhancements that opted in for docs created a placeholder PR? (anything below 90% opt-in for docs where placeholder PRs are needed should indicate yellow (or red) status, contingent on the amount)
```

[ ] A week before the Docs Ready for Review deadline, are 80% of the docs PRs ready for review? (anything below 60% should indicate yellow (or red) status, contingent on the amount)

> [ ] Three days before the Docs Ready for Review deadline, are 90% of the docs PRs ready for review?

Reviewer: ```suggestion
[ ] Three days before the Docs Ready for Review deadline, are 90% of the docs PRs ready for review? (anything below 90% of placeholder PRs being ready for review should indicate yellow (or red) status, contingent on the amount)
```

[ ] A week before the Docs Complete deadline, have 80% of the docs PRs merged? (anything below 60% should indicate yellow (or red) status, contingent on the amount)

> [ ] Three days before the Docs Complete deadline, have 90% of the docs PRs merged? (be sure to mention if you need the help of SIG Docs approvers/reviewers to meet your deadline)

Reviewer: Should the same guidance for red/yellow PRs be added to the final three items?

## Late Steps (Weeks 9-11) - Prep for the release

> The Docs Lead is responsible for updating the Kubernetes API reference, kubectl, and components documentation.

Reviewer 2: Since the reference docs has been generated by a SIG Docs TL since about 1.20 
How about  we replace the reference docs generation with a reminder or check-in with SIG Docs for the upcoming release and to prepare the reference docs generation

Author: Done!
